assessment_software/src/app/button-component/button.component.css

```
@@ -0,0 +1,15 @@
+.button-text {
+  font-size: 16px;
+  font-family: Inter;
+  font-weight: 500;
+  line-height: 22px;
+  word-wrap: break-word;
```

```suggestion
  font-size: 16px;
  font-family: Inter;
  font-weight: 500;
  line-height: 22px;
  word-wrap: break-word;
```@apply text-white bg-indigo-600 py-2.5 px-5 rounded-lg border-0 hover:bg-indigo-700;